### PR TITLE
design updated

### PR DESCRIPTION
**Pull request by Mansi168**
I have updated the design of website..please review

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Looks amazing, just require a small few changes.

- Get started button when clicked doesn't go down.
- Make the full backward text thing starting from "Type or Copy text here" a little up where you can see it with scrolling.
- Maybe change the fonts and colors?

Further, I can see that you added a image3.jpg in the assets folder, but it isn't used anywhere in the website?

Other than that, it's perfect

File: index.html [hunk 1]
```diff
@@ -10,14 +10,57 @@
...
   <div class="container">
     <!--- Code from https://www.roseindia.net/javascript/javascriptexamples/javascript-reverse-text-string.shtml -->
-    <h1 class="heading">REVERSE TEXT!</h1>
+    <h1 class="heading">Type or Copy text here</h1>
     <h3 class="sub-heading">
       You can reverse a name or any other text of your choice!
     </h3>
     <form onsubmit="stopRefresh(event)">
-      <input id="text" type="text" name="rev" />
+      <input  id="text" type="text" name="rev" />
```
[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply index.html]: @Mansi168 can you fix this?

**Comment by Mansi168** [comment]
> Looks amazing, just require a small few changes.
> 
> * Get started button when clicked doesn't go down.
> * Make the full backward text thing starting from "Type or Copy text here" a little up where you can see it with scrolling.
> * Maybe change the fonts and colors?
> 
> Further, I can see that you added a image3.jpg in the assets folder, but it isn't used anywhere in the website?
> 
> Other than that, it's perfect

okay,I'll make the changes asap.

**Comment by Mansi168** [comment]
yeah, i was adding image as background, but it was not looking good, that's why didn't use it

**Comment by KendallDoesCoding** [comment]
> yeah, i was adding image as background, but it was not looking good, that's why didn't use it

The gradient looks good, no worries.

**Comment by Mansi168** [comment]
@KendallDoesCoding , I have done the suggested changes. Please review.

**Review by KendallDoesCoding — APPROVED** [review]
Perfect. Will do small edits with the colors and fonts in a bit.

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution @Mansi168!!

**Pull request merged**